### PR TITLE
Issue 8 Highstepper Logo

### PR DESCRIPTION
**Pull request by adhamrait**
fixes issue #8

### CODE REVIEW

File: src/components/menu-top/MenuTop.vue [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <template>
   <div class="menu-top">
     <b-navbar type="is-primary">
+      <template slot="brand">
+        <b-navbar-item>
+          <img
+            src="/images/highstepper-gold.png"
```
Comment (MichaelTamaki) [inline src/components/menu-top/MenuTop.vue]: Sorry there’s no example for images, but can you take a look at using relative file paths instead? https://cli.vuejs.org/guide/html-and-static-assets.html#url-transform-rules
This way we won’t have conflicts when we deploy this to github pages (the file will be located at /calchart-redesign/images/highstepper-gold.png)
```suggestion
            src="@/images/highstepper-gold.png"
```

**Review by MichaelTamaki — CHANGES_REQUESTED** [review]
Can you place the image in src/assets instead? This way it goes through our webpack compiler. In the future, we will compress images that go through webpack.

File: src/components/menu-top/MenuTop.vue (hunk 1 above)
Comment (adhamrait) [reply src/components/menu-top/MenuTop.vue]: Changed!

**Review by MichaelTamaki — APPROVED** [review]
🚢🚢🚢Ship it!

**Pull request merged**